### DISCUSSION TITLE
I want pop up notification in directus - Is it possible?

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#15321] [Archive]
pop up notification in directus - Is it possible?

![image](https://user-images.githubusercontent.com/36895131/187468409-d248acc3-5714-4cd4-af2c-02be21758c73.png)

**Comment by rijkvanzanten** [comment]
Please don't open a Discussion for the same question multiple times 👍🏻 

https://github.com/directus/directus/discussions/15193

**Reply by Jeyakumarrathnaselvi** [reply]
both the queries are different #15193 is acitivity message but this one is popup

**Reply by rijkvanzanten** [reply]
Ah, gotcha

**Comment by rijkvanzanten** [comment]
Really depends on when and how you want to show that message. Any extension that renders using a Vue component can use the `<v-dialog>` element to render a dialog with any custom content for notices etc

**Reply by Jeyakumarrathnaselvi** [reply]
Thanks
